Source/Core/Updater/Main.cpp

```diff
@@ -670,6 +670,15 @@ void FatalError(const std::string& message)
 
 int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine, int nCmdShow)
 {
+  if (lstrlenW(pCmdLine) == 0)
+  {
+    MessageBox(nullptr,
+               L"This updater is not meant to be launched directly. Configure Auto-Update in "
+               "Dolphins Settings instead.",
```

Comment:
I believe that should be "Dolphin's"? Also, settings should probably not be capitalized.